kobo/django/xmlrpc/migrations/0001_initial.py

```diff
@@ -19,7 +19,8 @@ class Migration(migrations.Migration):
                 ('dt_inserted', models.DateTimeField(auto_now_add=True)),
                 ('method', models.CharField(max_length=255)),
                 ('args', models.TextField(blank=True)),
-                ('user', models.ForeignKey(blank=True, to=settings.AUTH_USER_MODEL, null=True)),
+                ('user', models.ForeignKey(blank=True, to=settings.AUTH_USER_MODEL, null=True,
+                                           on_delete=models.CASCADE)),
```

Comment:
I think this should be matching what we have in kobo/django/xmlrpc/models.py, shouldn't it?  That's on_delete=models.SET_NULL.

Reply:
Yeah, you're right. Thanks for pointing that out!